From aeb8c8c64be9e863f0e498c6b82e00528bbddfa7 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 17 Jul 2026 11:14:18 +0200
Subject: [PATCH] tests: py: Drop stale entries from ip6/{ct,meta}.t.json

JIRA: https://issues.redhat.com/browse/RHEL-190549
Upstream Status: nftables commit 3be0bbe9a5ba21a3f6e3baf051c3aaadc393c587

commit 3be0bbe9a5ba21a3f6e3baf051c3aaadc393c587
Author: Phil Sutter <phil@nwl.cc>
Date:   Wed Aug 13 15:50:54 2025 +0200

    tests: py: Drop stale entries from ip6/{ct,meta}.t.json

    Looks like these were added by accident, fixed commit did not add these
    test cases.

    Fixes: 8221d86e616bd ("tests: py: add test-cases for ct and packet mark payload expressions")
    Signed-off-by: Phil Sutter <phil@nwl.cc>
    Reviewed-by: Pablo Neira Ayuso <pablo@netfilter.org>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 tests/py/ip6/ct.t.json   | 164 ---------------------------------------
 tests/py/ip6/meta.t.json |  58 --------------
 2 files changed, 222 deletions(-)

diff --git a/tests/py/ip6/ct.t.json b/tests/py/ip6/ct.t.json
index 7d8c88b..a7511b9 100644
--- a/tests/py/ip6/ct.t.json
+++ b/tests/py/ip6/ct.t.json
@@ -1,167 +1,3 @@
-# ct mark set ip6 dscp lshift 2 or 0x10
-[
-    {
-        "mangle": {
-            "key": {
-                "ct": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "<<": [
-                            {
-                                "payload": {
-                                    "field": "dscp",
-                                    "protocol": "ip6"
-                                }
-                            },
-                            2
-                        ]
-                    },
-                    16
-                ]
-            }
-        }
-    }
-]
-
-# ct mark set ip6 dscp lshift 26 or 0x10
-[
-    {
-        "mangle": {
-            "key": {
-                "ct": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "<<": [
-                            {
-                                "payload": {
-                                    "field": "dscp",
-                                    "protocol": "ip6"
-                                }
-                            },
-                            26
-                        ]
-                    },
-                    16
-                ]
-            }
-        }
-    }
-]
-
-# ct mark set ip6 dscp << 2 | 0x10
-[
-    {
-        "mangle": {
-            "key": {
-                "ct": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "<<": [
-                            {
-                                "payload": {
-                                    "field": "dscp",
-                                    "protocol": "ip6"
-                                }
-                            },
-                            2
-                        ]
-                    },
-                    16
-                ]
-            }
-        }
-    }
-]
-
-# ct mark set ip6 dscp << 26 | 0x10
-[
-    {
-        "mangle": {
-            "key": {
-                "ct": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "<<": [
-                            {
-                                "payload": {
-                                    "field": "dscp",
-                                    "protocol": "ip6"
-                                }
-                            },
-                            26
-                        ]
-                    },
-                    16
-                ]
-            }
-        }
-    }
-]
-
-# ct mark set ip6 dscp | 0x04
-[
-    {
-        "mangle": {
-            "key": {
-                "ct": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "payload": {
-                            "field": "dscp",
-                            "protocol": "ip6"
-                        }
-                    },
-                    4
-                ]
-            }
-        }
-    }
-]
-
-# ct mark set ip6 dscp | 0xff000000
-[
-    {
-        "mangle": {
-            "key": {
-                "ct": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "payload": {
-                            "field": "dscp",
-                            "protocol": "ip6"
-                        }
-                    },
-                    4278190080
-                ]
-            }
-        }
-    }
-]
-
 # ct mark set ip6 dscp << 2 | 0x10
 [
     {
diff --git a/tests/py/ip6/meta.t.json b/tests/py/ip6/meta.t.json
index 1a2394d..87251f0 100644
--- a/tests/py/ip6/meta.t.json
+++ b/tests/py/ip6/meta.t.json
@@ -195,64 +195,6 @@
     }
 ]
 
-# meta mark set ip6 dscp lshift 2 or 0x10
-[
-    {
-        "mangle": {
-            "key": {
-                "meta": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "<<": [
-                            {
-                                "payload": {
-                                    "field": "dscp",
-                                    "protocol": "ip6"
-                                }
-                            },
-                            2
-                        ]
-                    },
-                    16
-                ]
-            }
-        }
-    }
-]
-
-# meta mark set ip6 dscp lshift 26 or 0x10
-[
-    {
-        "mangle": {
-            "key": {
-                "meta": {
-                    "key": "mark"
-                }
-            },
-            "value": {
-                "|": [
-                    {
-                        "<<": [
-                            {
-                                "payload": {
-                                    "field": "dscp",
-                                    "protocol": "ip6"
-                                }
-                            },
-                            26
-                        ]
-                    },
-                    16
-                ]
-            }
-        }
-    }
-]
-
 # meta mark set ip6 dscp << 2 | 0x10
 [
     {
